frontend/index.html

```diff
@@ -5,8 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Cyrillux - Tietueiden translitterointi Melindassa</title>
   
-  <link href='https://fonts.googleapis.com/css?family=Roboto+Mono' rel='stylesheet' type='text/css'>  
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Roboto+Mono' rel='stylesheet' type='text/css'> 
```

Comment:
Built the dependency into the bundle. See [Muuntaja](https://github.com/natlibfi/melinda-muuntaja) for refefence.